assessmentModel/src/androidLibMain/kotlin/android.kt

```diff
@@ -29,6 +29,6 @@ actual object UUIDGenerator {
 }
 
 actual object DateGenerator {
-    actual fun nowString(): String = "TODO: Implement"
+    actual fun nowString(): String = UUID.randomUUID().toString()   // TODO: syoung 06/16/2020 Replace with a timestamp using Android library
```

Comment:
I am using a UUID here b/c we don't yet have a solution for handling dates on Android. Using the UUID is a means of checking the unit tests so that they won't break once dates are implemented.

Reply:
I've had good luck with IslandTime for time/date/time-zone handling on both Android and iOS. Klock is another cross-platform library but IIRC serialization was broken with Kotlin Native 1.3.7X and there hasn't been an update

Reply:
The only feature I've found lacking in IslandTime so far is formatting/localization of the "Yesterday at 7:37am" or "Last Week" variety